Gemfile

```diff
@@ -1,8 +1,12 @@
 # frozen_string_literal: true
 
-source "https://rubygems.org"
+source 'https://rubygems.org'
+
 gemspec
 
-group :development do
-  gem "standard"
-end
+gem 'faraday', '~> 1.0'
```

Comment:
We should probably suggest keeping Faraday as an actual dependency in the gemspec. This was problematic for historical adapters, but it should not be a problem for new ones.

Also, we should suggest to pin `>= 1.10, < 3` so that the adapter will support both 1.x and 2.x.
A comment could explain that this would desirable, and if that's not possible (e.g. the adapter uses a new feature like the new streaming API), then a simple `~> 2.5` would suffice, although it might make it harder for users to use the adapter in legacy applications

Reply:
Alright, I agree, and I did that.